site-src/index.md

```diff
@@ -20,54 +20,119 @@ variety of networking use-cases.
 
 ![Gateway API Model](./images/api-model.png)
 
-
-How do Gateway API improve upon current standards like Ingress?
-
-- **More expressive** - They express more core functionality for things like
-header-based matching, traffic weighting, and other capabilities that were only
-possible in Ingress through custom means.
-- **More extensible** - They allow for custom resources to be linked at various
-layers of the API. This allows for more granular customization at the
-appropriate places within the API structure.
-- **Role oriented** - They are separated into different API resources that map
-to the common roles for running applications on Kubernetes.
-- **Generic** - This isn't an improvement but rather something
+Following are some of the design goals where Gateway aims to improve upon 
```

Comment:
Nit: I think it's still worth referring to this as "Gateway API" over Gateway in most docs just so we don't confuse users with the Gateway resource. That will likely still be somewhat confusing, but maybe slightly less so.

Reply:
yeah good point. I'll update API references to "Gateway API"